core/src/com/unciv/logic/automation/civilization/NextTurnAutomation.kt

```diff
@@ -129,15 +130,30 @@ object NextTurnAutomation {
 
     internal fun valueCityStateAlliance(civInfo: Civilization, cityState: Civilization, includeQuests: Boolean = false): Int {
         var value = 0
+        val civPersonality = civInfo.getPersonality()
 
-        if (civInfo.wantsToFocusOn(Victory.Focus.Culture) && cityState.cityStateFunctions.canProvideStat(Stat.Culture)) {
-            value += 10
+        if (cityState.cityStateFunctions.canProvideStat(Stat.Culture)) {
+            if (civInfo.wantsToFocusOn(Victory.Focus.Culture))
+                value += 10
+            value += civPersonality[PersonalityValue.Culture].toInt() - 5
```

Comment:
Can we change these `civPersonality[PersonalityValue.Culture].toInt() - 5` lines to use something like `civPersonality.getNormalizedValue(range = 5)` to be more readable. The important thing is the range parameter shows here what values it will give out for readability.

Reply:
I know Yarim mentioned something similar, but my issue is kinda simple: wouldn't that read as `civPersonality.getNormalizedValue(PersonalityValue.Culture, range = 5)`. Which means
1. There's no real shortening of lines, nor any real simplification. If anything, it makes lines longer
2. It's only better for readability if you know what a "nornalizedValue" should mean in this case. Sure, kdocs, but I'm pretty sure we could address questions here with kdocs as well
3. You're suggesting a more expensive version of `.get` or `(.scaledFocus * 5).toInt()`... For just this?
4. Why not just create an overload for `scaledFocus`? I'm all for doing this, but then my question is "why are we using that as a replacement for square brackets/.get"

Reply:
>  Why not just create an overload for scaledFocus? I'm all for doing this, but then my question is "why are we using that as a replacement for square brackets/.get"

The main goal is abstraction, encapsulation and lower coupling. If I am accessing a certain civPersonality in some other file, the reader shouldn't have to know about how the actual personality data structure is implemented. (ie: is it stored in a hashmap and what is the range of the personality values?) It doesn't have to be shorter and the performance cost here is not an issue.
 
Also `(range = 5)` and `getNormalizedValue` were just a suggestion.

Reply:
Ok, but I'm not sure where the benefits of encapsulation is here, especially since there already is a getter/setter that already handles things pretty cleanly. Like, the code will always have to read ``persobslity.getterFunction(PersonalityValue, whateverArgument)``, with the `toInt` likely being ideal as well

Again, I can totally seeing adjusting `scaledFocus` to cover what you want. But the only thing your suggestion addresses assuming the scale is 0-10 vs any other set of values... Which shouldn't matter here anyways because it's still going to be communicated to the modder as 0-10, and this usecase in particular is still using specifically that assumption